### PR TITLE
[Merged by Bors] - beacon: add public metrics for current and next epoch

### PR DESCRIPTION
**Pull request by countvonzero**

### CODE REVIEW

**Review by fasmat — APPROVED** [review]

File: metrics/public/public.go [hunk 1]
```diff
@@ -26,6 +26,14 @@ var (
...
 		Name:      "version",
 	}, []string{"version"})
+	CurrentBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "smh",
+		Name:      "beacon",
+	}, []string{"curr_beacon"})
+	NextBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "smh",
+		Name:      "next_beacon",
+	}, []string{"beacon"})
```
Comment (poszu) [inline metrics/public/public.go]: How about
```suggestion
	CurrentBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
		Namespace: "smh",
		Subset:    "beacon",
		Name:      "current,
	}, []string{"beacon"})
	NextBeacon = promauto.With(Registry).NewGaugeVec(prometheus.GaugeOpts{
		Namespace: "smh",
		Subset:    "beacon",
		Name:      "next",
	}, []string{"beacon"})
```

to get metrics:
- `smh_beacon_current{beacon=}`
- `smh_beacon_next{beacon=}`

**Review by poszu — APPROVED** [review]

**Comment by countvonzero** [comment]
bors merge

**Comment by countvonzero** [comment]
bors cancel

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**